### DISCUSSION TITLE
Font size

### DISCUSSION THREAD

**Discussion by michaelzolo** [gitbito/bitoai#152] [Q&A]
I haven't found a way to increase the size of the font in the plugin. Changing editor's or menu's fonts has no effect.

**Comment by SharukaV** [comment]
That's right.
I even tried to fix the source code, but I was unsuccessful.
It is after all even discriminating against people who are visually impaired or not so young!

**Comment by SharukaV** [comment]
This one worked a little bit for me
![Monosnap Settings 2023-12-13 15 37 14](https://github.com/gitbito/bitoai/assets/46028834/3ae1f595-21f9-4c03-bca8-aa215c88e8cb)

**Comment by wxwok** [comment]
The text line spacing of the plugin is also too small in the CLion new UI.
![image](https://github.com/gitbito/bitoai/assets/561843/506e3511-fdd5-4c94-a59a-1c475d0e98cf)
